server/config/database_config.js

```diff
@@ -0,0 +1,32 @@
+import path from 'path';
+import { config } from 'dotenv';
+
+config();
+
+export const development = {
+  username: process.env.DB_USERNAME,
+  password: process.env.DB_PASSWORD || '',
+  database: process.env.DB_DATABASE,
+  host: process.env.DB_HOST,
+  dialect: 'postgres'
+};
+
+export const test = {
+  username: process.env.DB_USERNAME || 'postgres',
```

Comment:
remove this

The `sqlite` should not need `username` `password` etc

`dialect` and `storage` should be enough